### ISSUE TITLE
Wrong OneDrive sync version reference in section "Silently move Windows known folders to OneDrive"

### ISSUE THREAD

**Issue by soljan** [MicrosoftDocs/OfficeDocs-SharePoint#282]
Under "Silently move Windows known folders to OneDrive" there's a reference to OneDrive "sync client build 18.171.0823.0001 or later", it should be "18.111.0603.0004 or later". 

---
#### Dokumentinformasjon

⚠ *Ikke rediger denne delen. Den kreves for koblingen docs.microsoft.com ➟ GitHub-problem.*

* ID: 04ad2f16-c861-7a1b-5c1d-e4601e8f3855
* Version Independent ID: c47df4cd-f706-d807-5eac-172fba1e500f
* Content: [Use Group Policy to control OneDrive sync client settings](https://docs.microsoft.com/nb-no/onedrive/use-group-policy#silently-move-windows-known-folders-to-onedrive)
* Content Source: [OneDrive/use-group-policy.md](https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/blob/live/OneDrive/use-group-policy.md)
* Service: **one-drive**
* GitHub Login: @kaarins
* Microsoft Alias: **kaarins**

**Comment by AndreaBarr** [comment]
@soljan thank you for this feedback

Yes we are waiting on the confirmation and updated information so that I can update that section of the article. We apricate you pointing out that the version is incorrect. 

I was able to get this into the backlog for the SharePoint team. I am going to close this issue so we don't have two tickets open for this. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated. 
 
I made a note to request the team to update this when the work is complete.
